utils/src/main/scala/za/co/absa/enceladus/utils/fs/example/S3FsUtilsTestJob.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.utils.fs.example
+
+import org.apache.spark.sql.SparkSession
+import org.slf4j.LoggerFactory
+import software.amazon.awssdk.regions.Region
+import za.co.absa.atum.persistence.S3KmsSettings
+import za.co.absa.atum.utils.S3Utils
+import za.co.absa.enceladus.utils.fs.S3FsUtils
+
+// todo remove or create a integtest like this instead.
+// implementation is directly suited to be runnable locally with a saml profile.
+object S3FsUtilsTestJob {
```

Comment:
This entrypoint is entirely designed for testrunning the S3FsUtils on local machine against a real s3 using the local saml profile. Future integTest can be heavily inspired by this class, so maybe it could stay until then?

Reply:
I would say it can stay, but use different paths in s3

Reply:
I have changed the base-path, what is left can reasonably serve as a generic example.